guide/case-studies/payjoin.md

```diff
@@ -0,0 +1,333 @@
+      caption: A transition screen is displayed indicating that payjoin construction and signing process.
+    - file: send-review
+      modalImage: send-review-big
+      alt: A pre-sending review screen showing the finalised fee after the payjoin handshake is completed.
+      caption: The pre-broadcast review screen with the finalised fee after the payjoin handshake is completed.
+    - file: send-payjoin-success
+      modalImage: send-payjoin-success-big
+      alt: A payment success screen informs the user that the Payjoin transaction has been broadcast.
+      caption: A payment success screen informs the user that the Payjoin transaction has been broadcast.
+
+---
+
+<!--
+
+Editor's notes
+
+Illustration sources: <>
+
+-->
+
+{% include picture.html
+   image = "assets/images/guide/case-studies/payjoin/header-graphic-metaphor-banner.png"
+   retina = "assets/images/guide/case-studies/payjoin/header-graphic-metaphor-banner@2x.png"
+   alt-text = "Payjoin case study banner image"
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+
+# The Payjoin Experience
+{:.no_toc}
+---
+
+<div class="glossary-toc" markdown="1">
+* Table of contents
+{:toc}
+</div>
+
+---
+
+A bitcoin transaction where the sender and receiver collaboratively construct a transaction is called a Payjoin. Payjoin transactions can be a powerful tool providing privacy, fee-savings and UTXO consolidation benefits to users. They also present challenges and trade-offs due to their interactive & synchronous nature.
+
+**Payjoin transactions break some of the chain analysis [heuristics](https://github.com/bitcoin/bips/blob/master/bip-0078.mediawiki#motivation) during the course of making a payment. However, real time coordination between the sender & receiver mediated through an endpoint is required to construct them.**
+
+This case study:
+
+- Outlines the incentives and objectives for both senders & receivers
+- Analyses the Payjoin UX on both sides and proposes approaches to design Payjoin experiences.
+- Proposes specific user flows and where possible, working designs for the same.
+
+The high-level findings from this case study are that while the setup is more involved for the receiver, little-to-no effort is required per transaction.
+
+The sender on the other hand needs little-to-no setup (apart from using a supporting wallet) and can achieve an identical or near-identical user experience to a regular transaction, depending on their choices and the sophistication of the implementation.
+
+
+### What is a Payjoin?
+
+In a payjoin, both the sender and receiver contribute inputs to the transaction in a coordinated manner. The payjoin mechanism is also called pay-to-endpoint (P2EP) since the coordination is mediated through an endpoint run by the receiver. The endpoint address is communicated through a [BIP-21](https://bitcoinqr.dev) URI along with the payment address and amount.
```

Comment:
```suggestion
In a payjoin, both the sender and receiver contribute inputs to the transaction in a coordinated manner. The payjoin mechanism is also called pay-to-endpoint (P2EP), since the coordination is mediated through an endpoint run by the receiver. The endpoint address is communicated through a [BIP-21](https://bitcoinqr.dev) URI, along with the payment address and amount.
```